style.css

```diff
@@ -0,0 +1,192 @@
+
+body {
+  background: #eee;
+}
+
+#header {
+  display: flex;
+  justify-content: space-between;
+  position: fixed;
+  width: 100%;
+  top: 0;
+  right: 0;
+  left: 0;
+  height: auto;
+  z-index: 1;
+  background-color: #eee;
+}
+
+#header-img {
+  width: 300px;
+  font-family: 'Lato', sans-serif;
+  padding: 20px;
+  height: 35px;
+}
+
+header nav {
+  display: flex;
+  line-height: 60px;
+  padding-right: 30px;
+}
+
+main h1 {
+  margin: 0 auto;
+  padding: 10px 0;
+}
+
+main form {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  align-items: center;
+}
+
+#email {
+  width: 350px;
+  padding: 7px;
+  margin: 12px;
+  border: 0.5px solid black;
+}
+
+#submit {
+  width: 180px;
+  padding: 8px 13px;
+  margin: 5px 0 20px;
+  background-color: rgb(233, 197, 35);
+  border-radius: 0;
+  border: 1px none transparent;
+  font-weight: 700;
+  font-size: 18px;
+}
+
+section article {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  width: 700px;
+  padding: 0;
+  margin: 0 auto 15px;
+}
+
+section article .article-bar {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+  width: 70%;
+}
+
+section article .article-bar h1 {
+  padding: 7px 0;
+  margin: 0;
+  font-size: 21px;
+}
+
+section article .article-bar p {
+  padding: 5px 0;
+  margin: 0;
+}
+
+iframe {
+  width: 580px;
+  height: 300px;
+  margin: auto;
+  padding: 30px 0;
+  display: flex;
+  justify-content: center;
+}
+
+.articles {
+  display: flex;
+  flex-wrap: wrap;
+  width: 50%;
+  padding: 0;
+  margin: auto;
+}
+
+.articles .side-band {
+  display: flex;
```

Comment:
**Hi VTY1999**
Great work so far, You used display: flax in the side band but i would have preferred it if you us float: left so it is more organized.
```suggestion
  float: left;
```